src/wallet/types.ts

```diff
@@ -449,14 +454,16 @@ export interface TxByIdTokenData {
   voided: boolean;
   height?: number | null;
   weight: number;
-  balance: Balance;
```

Comment:
Type is wrong, it never came as a Balance object from the wallet-service:

https://github.com/HathorNetwork/hathor-wallet-service/blob/239b5a769b04a09714ca7fa673e99e064e91f471/packages/wallet-service/src/db/index.ts#L2963